tests/integration/dual-channel-replication.tcl

```diff
@@ -1375,3 +1375,70 @@ start_server {tags {"dual-channel-replication external:skip"}} {
         }
     }
 }
+
+test "Chained replicas does not assert when using dual channel replication" {
+    start_server {tags {"dual-channel-replication external:skip"}} {
+        set primary [srv 0 client]
+        set primary_host [srv 0 host]
+        set primary_port [srv 0 port]
+        set primary_pid [srv 0 pid]
+
+        $primary config set repl-diskless-sync yes
+        $primary config set dual-channel-replication-enabled yes
+        $primary debug pause-after-fork 1
+        $primary debug delay-rdb-client-free-seconds 60
+
+        start_server {} {
+            set replica [srv 0 client]
+            set replica_host [srv 0 host]
+            set replica_port [srv 0 port]
+            set replica_pid  [srv 0 pid]
+            set loglines [count_log_lines 0]
+
+            $replica config set dual-channel-replication-enabled yes
+
+            test "Psync established after rdb load - within grace period" {
```

Comment:
Nested `test`? Usually we don't do that. Let's do only `start_server` on the top level.

Reply:
yes, i forgot to remove it, this is a pre-test i copied